apps/server/src/app/export/msword.ts

```diff
@@ -40,6 +58,225 @@ export class MSWordExporter {
+          }
+
+
+          if (structureDefinition.differential) {
+            (structureDefinition.differential.element || []).forEach((element, eIndex) => {
+                if(element.id === 'Extension'){
+                  this.body.push(new Paragraph({ text: `Element ${eIndex + 1}` }));
+                  this.body.push(new Paragraph({ text: `Id ${element.id}` }));
+                  this.body.push(new Paragraph({ text: `Path: ${element.path}` }));
+                }
+                if(element.id === 'Extension.url') {
+                  this.body.push(new Paragraph({ text: `Element ${eIndex + 1}` }));
+                  this.body.push(new Paragraph({ text: `Extension.url ${element.id}` }));
+                  this.body.push(new Paragraph({ text: `Path: ${element.path}` }));
+                }
+                if(element.id === 'Extension.value[x]') {
+                  this.body.push(new Paragraph({ text: `Element ${eIndex + 1}` }));
+                  this.body.push(new Paragraph({ text: `Extension.value[x] ${element.id}` }));
+                  this.body.push(new Paragraph({ text: `Path: ${element.path}` }));
+                }
+                this.body.push(new Paragraph({ text: `Fixed: ${element.fixed}` }));
+                this.body.push(new Paragraph({ text: `Min: ${element.min}` }));
+                this.body.push(new Paragraph({ text: `Max: ${element.max}` }));
+                this.body.push(new Paragraph({ text: `Must support: ${element.mustSupport}` }));
+                this.body.push(new Paragraph({ text: `Binding: ${element.binding}` }));
+                this.body.push(new Paragraph({ text: `Short: ${element.short}` }));
+                this.body.push(new Paragraph({ text: `Label: ${element.label}` }));
+
+              }
+            );
+          };
+
+
+
+            keys.forEach((key) => {
+            //if(key === 'name' || 'url' || 'description'){
+            const table = new Table({
+              rows: [
+                new TableRow({
+                  children: [
+                    new TableCell({
+                      children: [new Paragraph(key)],
+                      verticalAlign: VerticalAlign.CENTER,
+                    }),
+                    new TableCell({
+                      children: [new Paragraph(entry.resource[key])],
+                      verticalAlign: VerticalAlign.CENTER,
+                    }),
+                  ]
+                })
+              ]
+            });
+            this.body.push(table);
+          //}
+          });
+
+          this.body.push(new Paragraph({
+            text: ""
+          }));
+          }
+        );
+
+      bundle.entry
+        .filter(entry => entry.resource && (entry.resource.resourceType === 'ValueSet'))
+        .forEach((entry) => {
+            const extensionUrlKeys = Object.keys(Globals.extensionUrls);
```

Comment:
I'd rather output a select number of properties, rather than just looping through all the keys...
* name
* title
* description
* purpose
* copyright
* compose
  * include
    * system
    * concept
      * code
      * display
    * filter
      * property, op, value
  * exclude
    * system
    * concept
      * code
      * display
    * filter
      * property, op, value